windows/src/engine/keyman32/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+builder_describe "Keystroke processing engine (32 bit)" \
+  @/common/include \
+  @/core:x86 \
+  clean configure build test publish install
+
+builder_parse "$@"
+
+#-------------------------------------------------------------------------------------------------------------------
+
+source "$KEYMAN_ROOT/resources/build/win/environment.inc.sh"
+WIN32_TARGET="$WIN32_TARGET_PATH/keyman32.dll"
+
+builder_describe_outputs \
+  configure:project    /resources/build/win/delphi_environment_generated.inc.sh \
+  build:project        /windows/src/engine/keyman32/$WIN32_TARGET
+
+#-------------------------------------------------------------------------------------------------------------------
+
+function do_clean() {
+  vs_msbuild keyman32.vcxproj //t:Clean
+  clean_windows_project_files
+}
+
+function do_build() {
+  create-windows-output-folders
+  build_version.res
+  vs_msbuild keyman32.vcxproj //t:Build "//p:Platform=Win32"
+  cp "$WIN32_TARGET" "$WINDOWS_PROGRAM_ENGINE"
+  cp "$WIN32_TARGET_PATH/keyman32.pdb" "$WINDOWS_DEBUGPATH_ENGINE"
+}
+
+function do_publish() {
+  wrap-signcode //d "Keyman Engine for Windows" "$WINDOWS_PROGRAM_ENGINE/keyman32.dll"
+  wrap-symstore "$WINDOWS_PROGRAM_ENGINE/keyman32.dll" //t keyman-engine-windows
```

Comment:
Just querying this change in behaviour 
for my local publish this gets skipped as $KEYMAN_SYMSTOREPATH is not is set an so wrap-symstore exits early. Previously the Defines.mak file had this line.
# KEYMAN_SYMSTOREPATH defaults to sibling folder "symbols". If it is not present,
# then we won't attempt to write symbols to the store.
```
!IFNDEF KEYMAN_SYMSTOREPATH
KEYMAN_SYMSTOREPATH=$(KEYMAN_ROOT)\..\symbols
!ENDIF
```

Reply:
I will add a check to wrap-symstore.

Reply:
It's already there, see:

https://github.com/keymanapp/keyman/blob/8b2070c339a0ec424ae5d7a9fcbb23e97d8afc6a/resources/build/win/environment.inc.sh#L117-L121